dcos_installer/test_backend.py

```diff
@@ -297,54 +274,50 @@ def test_do_aws_configure(tmpdir, monkeypatch):
         assert backend.do_aws_cf_configure() == 0
 
 
-valid_storage_config = """---
+@pytest.fixture
+def valid_storage_config(release_config_aws):
+    """ Uses the settings from dcos-release.config.yaml ['testing'] to create a
+    new upload and then deletes it when the test is over
+    """
+    s3_bucket_name = release_config_aws['bucket']
```

Comment:
`release_config_aws.get('bucket', '')` ?  Or do you want this to fail with `KeyError` ?

Reply:
You most definitely want this to fail with a key error. Running this without a pre-configured bucket for writing to not a valid test case (as the advanced aws template doesnt itself create a new bucket, but rather just a bucket path)